GamePortal/GamePortal.Web.Api/App_Start/NinjectWebCommon.cs

```diff
@@ -87,8 +88,8 @@ private static void RegisterServices(IKernel kernel)
                 .MinimumLevel.Verbose()
                 .CreateLogger();
 
-            kernel.Bind<ILogger>().ToConstant(logger);
-            kernel.Load(/*new LogicDIModule(),*/ new TTGDIModule()/*, new BattleshipLogicDIModule()*/);
+            kernel.Bind<ILogger>().ToConstant(TTGlogger);
```

Comment:
split your logger with db writing from another. you use your private db for logging.